src/com/zoffcc/applications/CustomWidgetProvider.java

```diff
@@ -0,0 +1,57 @@
+package com.zoffcc.applications;
+
+import android.app.PendingIntent;
+import android.appwidget.AppWidgetManager;
+import android.appwidget.AppWidgetProvider;
+import android.content.Context;
+import android.content.Intent;
+import android.view.View;
+import android.widget.RemoteViews;
+
+import org.smssecure.smssecure.R;
+
+/**
+ * Created on 01.09.2016.
+ */
+public class CustomWidgetProvider extends AppWidgetProvider
+{
+
+    // ---- CONFIG ----
+    // ---- CONFIG ----
+    // ---- CONFIG ----
+    public static final int MAX_COUNT = 20;
```

Comment:
suggestion: set MAX_COUNT value at least to 99, it's still 2-digit number (better 999 if possible)